### ISSUE TITLE
Stuck at (match_worker_ray_wrapper pid=32734) Global seed set to 6660/725 [00:00<?, ?it/s] (match_worker_ray_wrapper pid=32736) Global seed set to 666 (match_worker_ray_wrapper pid=32735) Global seed set to 666 (match_worker_ray_wrapper pid=32765) Global seed set to 666

### ISSUE THREAD

**Issue by brian2lee** [zju3dv/OnePose_Plus_Plus#40]
As the title mention, I got stuck when running `bash scripts/demo_pipeline.sh $OBJ_NAME`. After trying the methods mention in this [issue](https://github.com/zju3dv/OnePose_Plus_Plus/issues/15) it got through but showed a new error `(match_worker_ray_wrapper pid=27622) NVIDIA GeForce RTX 3060 Laptop GPU with CUDA capability sm_86 is not compatible with the current PyTorch installation.
` in an attempt to solve it I install other version of pytorch using`mamba install -c pytorch -c conda-forge pytorch=1.10.0 torchvision cudatoolkit=11.3 --yes`. Which got me back to stuck at Global seed 666.
In conclusion, either I''ll got stuck on Global seed 666 or CUDA capability. Any way to solve this problem?

**Comment by lejy026** [comment]
Do you solve it ?

**Comment by lejy026** [comment]
This issue maybe can be solved by  add weight/LoFTR_wsize9.ckpt

**Comment by anshium** [comment]
#5
